src/test/admin_socket.cc

```diff
@@ -140,11 +140,12 @@ class MyTest : public AdminSocketHook {
 TEST(AdminSocket, RegisterCommand) {
   std::unique_ptr<AdminSocket>
       asokc(new AdminSocket(g_ceph_context));
+  std::unique_ptr<AdminSocketHook> test_hook(new MyTest());
```

Comment:
use `std::make_unique<>` if you please.

Reply:
using `std::make_unique<>` is a good idea! and i have changed `test_hook` to `my_test_asok`, here asok is short for **a**dmin **s**ocket ho**ok**, since `my_test_asok` is an instance of `MyTest`, I think it is a better name than asokc_hook, what do you think?

Reply:
> since `my_test_asok` is an instance of `MyTest`

right, but ```class MyTest : public AdminSocketHook```. anyway, I don't feel strong either way. just thought that "test" is too general in the context of unit test.